packages/near-membrane-base/src/membrane.ts

```diff
@@ -1224,6 +1168,13 @@ export function init(
                 throw pushErrorAcrossBoundary(e);
             }
         },
+        // callableLinkPointers: this callable function allows the foreign
+        // realm to define a linkage between two values across the membrane.
+        (targetPointer: Pointer, newPointer: Pointer) => {
```

Comment:
this callable is basically the hook to add things into the weakmap that reflects objects from one to the other.